README.md

```diff
@@ -2,56 +2,80 @@
+2. Plugins (SDK): Various features like checking node status, collecting node metrics and executing certain commands, can be integrated to VATZ through separate plugins. 
+3. Monitoring: Various logs and data of nodes are exported by a node exporter and monitored through the 3rd party applications like Graphana. 
 
-We're on a mission to transform the way people experience blockchain technology and let them contribute and become a part of its technology.
-As Validators, we provide tools to people to manage their own nodes with low cost and less effort for anyone who would like to join future blockchain technology.
+For further information, check [VATZ Project Design](docs/design.md)
 
 
-## 3. Contact
-Please, contact [us](mailto:validator@dsrvlabs.com) if you need any further information about **VATZ**. <br> Visit [DSRV Labs](https://www.dsrvlabs.com/) if you are interested more about what we are doing
+## What is the key feature of **VATZ Project**?
 
-[<img alt="Medium" src="https://user-images.githubusercontent.com/6308023/176984456-f82c5c67-ebf3-455c-8494-c64ebfd66c58.svg" width="50" height="50" />](https://dsrv.medium.com)
-[<img alt="Github" src="https://user-images.githubusercontent.com/6308023/176984452-c73aa188-563a-4b93-8ad8-cd7974770275.svg" width="50" height="50" />](https://github.com/dsrvlabs)
-[<img alt="Facebook" src="https://user-images.githubusercontent.com/6308023/176984453-2fc6cd38-b0f8-4c44-a8d5-82291b3ce4f2.svg" width="50" height="50" />](https://www.facebook.com/dsrvlabs)
-[<img alt="Youtube" src="https://user-images.githubusercontent.com/6308023/176984454-52c20db5-6b8f-4c15-a621-dd4a0052e99f.svg" width="50" height="50" />](https://www.youtube.com/channel/UCWhv8Kd430cEMpEYBPtSPjA/featured)
-[<img alt="Twitter" src="https://user-images.githubusercontent.com/6308023/176984455-d48b24a9-1eb4-4c38-b728-2f4a0ccff09b.svg" width="50" height="50" />](https://twitter.com/dsrvlabs)
+  ### Multi Protocol Support
+   **VATZ** is NOT limited Protocol Type where it categorizes on chain protocol. Any Protocol can be managed through VATZ with plugins, even unsupported protocols can be integrated through simple plugin development.
+  ### Infrastructure as Code
+  **VATZ** is described using a high-level configuration syntax. You can divide your plugins into modular components that can then be combined in different ways to behave through automation.
+  ### Data Analysis
+   **VATZ** helps to build datasets for your managing protocols and transfer your data into popular services Prometheus, Kafaka, Google BigQuery and more. Because of this, VATZ aims to set your Node infrastructure as efficiently as possible, and operators get insight into dependencies in their infrastructure. (2023-Q3)
+  ### Change automation
+  Complex sets of node's operational tasks can be done through **VATZ** with minimal human interaction. (2023-Q2)
```

Comment:
Can you push this to Q4

Reply:
data analysis and automation both?

Reply:
Data Analysis -> 2023 Q3
Change automation -> 2023 Q4

Reply:
updated!